minion/client/__main__.py

```diff
@@ -43,6 +43,10 @@ def init():
     shutil.copyfile(os.path.join(os.path.dirname(__file__), "templates/minion.yml"), "minion.yml")
     click.echo("Added 'git minion' alias")
 
+@cli.command()
+def stop():
```

Comment:
You can use `@click.pass_context` decorator to pass the `ctx` argument to the CLI function. This object is initialized on line 24 - it contains a client connection to the minion server.

In the `stop` function you can then use `ctx.obj.stop_server()`